azerothcore/azerothcore-wotlk
# fix(DB/GO): Makes two mining Nodes wholly visible

## Changes Proposed:
-  Slightly changes the position of two mining Nodes, so that they are not "stuck" in the ground and wholly visible

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7759

## SOURCE:


## Tests Performed:

- Ran SQL, no errors
- checked in-game


## How to Test the Changes:


1. `.go o 15462` check that it is visible
2. `.go o 15423` check that it is visible

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
